configs/components/leatherman.json

```diff
@@ -0,0 +1 @@
+{"url": "git://github.com/puppetlasb/leatherman.git", "ref": "0.3.2"}
```

Comment:
Now that we have a tagged leatherman that works standalone, I've pinned this to that tag